event/listener.php

```diff
@@ -111,6 +112,22 @@ public function do_poll_modification($event)
 	}
 
 	/**
+	 * Modifys the template vars to match the advanced poll settings
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function do_poll_block_modification($event)
+	{
+		if ($event['s_display_results'] && (int) $event['topic_data']['wolfsblvt_poll_show_ordered'])
```

Comment:
I have looked into this further, and I have pushed this check into the advancedpolls class, while I keep here the same check that you have for the other cases, as you suggested.
The issue is already created.